src/main/java/com/softserve/academy/spaced/repetition/dto/impl/PasswordDTO.java

```diff
@@ -26,8 +26,10 @@ public boolean equals(Object o) {
         if (!(o instanceof PasswordDTO)) return false;
```

Comment:
Sonar should report that curly brackets are missing.

Reply:
Sonar want: Replace this if-then-else statement by a single return statement. Is it need to do in one if statement?

Reply:
You are talking about line 29. https://sbforge.org/sonar/rules/show/squid:S1126?layout=false

Reply:
Looks like curly brackets are not forced by Sonar, it is a rule on my current project.

Reply:
Yes I about it, so I fix curly brackets